lavamoat/browserify/main/policy.json

```diff
@@ -1352,14 +1354,6 @@
         "@metamask/keyring-api>bech32": true
       }
     },
-    "@metamask/profile-sync-controller>@metamask/keyring-api": {
-      "packages": {
-        "@metamask/keyring-api>@metamask/keyring-utils": true,
-        "@metamask/utils>@metamask/superstruct": true,
-        "@metamask/profile-sync-controller>@metamask/keyring-api>@metamask/utils": true,
-        "@metamask/keyring-api>bech32": true
-      }
-    },
```

Comment:
Not sure why this is being removed

Reply:
Most probably because on `main` the `keyring-api` version used by `profile-sync-controller` is different from the one being used by the extension.

On `main`:
```console
$ yarn why @metamask/keyring-api -R
└─ metamask-crx@workspace:.
   ├─ @metamask/keyring-api@npm:12.0.0 (via npm:^12.0.0)
   ...
   └─ @metamask/profile-sync-controller@npm:4.1.0 [15918] (via npm:^4.1.0 [15918])
      └─ @metamask/keyring-api@npm:13.0.0 (via npm:^13.0.0)
```

Reply:
didn't know it was version 13.0.0 for `profile-sync-controller`. Thanks for checking